frontend/privacypolicy.html

```diff
@@ -0,0 +1,92 @@
+<!-- <!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Privacy Policy</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Kode+Mono:wght@400..700&display=swap" rel="stylesheet">
+
+	<script type="module" src="./static/js/index.js" defer></script>
+	<link rel="stylesheet" href="./static/css/styles.css">
```

Comment:
Do we need to keep these comments here? I am not familiar with how html pages conventionally are written, do they have a head an some references commented out?

Reply:
No need to keep, I thought I deleted those :D I'll make the change!